### DISCUSSION TITLE
目前git一个2.10.1版本的，今天一看官方有提交了一些东西，我这边如何更新

### DISCUSSION THREAD

**Discussion by jianghang1995** [vbenjs/vue-vben-admin#3390] [General]
我git 然后上来就给.git 文件删除了，使用git init 后 git remote add vben https://github.com/vbenjs/vue-vben-admin.git 后 git pull vben main 出现了错误，请问这些怎么更新

**Comment by Deeplycovered-leaf** [comment]
请问你后来是怎么更新的

**Reply by dingdayu** [reply]
哈哈，我是 Git Merge
